src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -18,6 +19,8 @@
 
 #include <lib/SecondaryMouseButtonsHook.h>
 
+extern "C" IMAGE_DOS_HEADER __ImageBase;

```

Comment:
What is this for?

Reply:
It's needed for `GET_RESOURCE_STRING` macro
check the comment here 
https://github.com/microsoft/PowerToys/blob/master/src/common/common.h#L73